backend/db/ad_hoc_omop_vocab_version.py

```diff
@@ -0,0 +1,58 @@
+"""Ad hoc script: to populate omopVocabVersion in the database
+
+Inteded for single use, 2023/11/07"""
+from datetime import datetime
+from typing import Dict, List
+
+from jinja2 import Template
+
+from backend.db.utils import get_db_connection, refresh_any_dependent_tables, run_sql
+from enclave_wrangler.objects_api import fetch_all_csets
+
+
+# COALESCE was another option. What is the advantage?
+sql_jinja = """
+UPDATE n3c.code_sets
+SET omop_vocab_version = CASE
+    {% for k, v in key_vals.items() %}
+      WHEN codeset_id = {{k}} THEN '{{v}}'
+    {% endfor %}
+END
+WHERE codeset_id IN ({{id_list}});"""
+
+
+def populate_omop_vocab_version(update_code_sets=False, update_derived_tables=False):
+    """Populate omopVocabVersion in the database.
+
+    Since this is an ad hoc script meant only to be run once, defaulted params to False after successfully running.
+    :param update_code_sets (bool): Update the code_sets table.
+    :param update_derived_tables (bool): Updates derived tables."""
+    if update_code_sets:
+        print('Fetching all csets')  # 15 seconds
+        t0 = datetime.now()
+        csets: List[Dict] = fetch_all_csets()
+        key_vals = {x['codesetId']: x['omopVocabVersion'] for x in csets}
+        t1 = datetime.now()
+        print(f' - completed in {(t1 - t0).seconds} seconds')
+
+        print('Updating code_sets')  # 2 seconds
```

Comment:
## Updated `code_sets`
`code_sets` has 7,392 entries. I got 7,186 entries got back from enclave after filtering out archived cset containers and source app 1.0 csets.

So after doing this update, there are 205 csets there where `omop_vocab_version = NULL`: 
[remaining null omop_vocab_version after updating all fetchable csets.csv.zip](https://github.com/jhu-bids/TermHub/files/13291701/remaining.null.omop_vocab_version.after.updating.all.fetchable.csets.csv.zip)

I'm surprised this number is not 206 (7392 - 7186). But I imagine there's a good reason; perhaps we had already set the version for one of these csets and it has since been archived.